packages/sources/mycryptoapi/README.md

```diff
@@ -0,0 +1,54 @@
+# Chainlink External Adapter for MyCryptoApi
+
+### Environment Variables
+
+| Required? |    Name     |             Description              |  Options   | Defaults to |
+| :-------: | :---------: | :----------------------------------: | :--------: | :---------: |
+|           | API_VERBOSE | enable/disable verbose output format | true/false |             |
```

Comment:
```suggestion
```

We don't have to document this env var here, as it's a standard one across all EAs.